lib/build-view.js

```diff
@@ -158,10 +162,16 @@ export default class BuildView extends View {
     if (0 === w || 0 === h) {
       return;
     }
-
+    const orientation = atom.config.get('build.panelOrientation') || 'Bottom';
+    if ( orientation === 'Left' || orientation === 'Right' ) {
+      this.cache[orientation].width = $('.build .output').width();
+    } else if ( orientation === 'Top' || orientation === 'Bottom' ) {
+      this.cache[orientation].height = $('.build .output').height();
```

Comment:
Previously the height was taken from  `this.terminalEl`. This is not equivalent if `terminalEl` has margin, borders or padding. I think this should still grab the height from `this.terminalEl`, or calculations will off.

Was this change the thing that actually solved the issue?